pkg/server/structlogging/hot_ranges_log.go

```diff
@@ -24,6 +24,13 @@ import (
 // ReportTopHottestRanges limits the number of ranges to be reported per iteration
 const ReportTopHottestRanges = 5
 
+// HotRangeLogManualTicker is a channel that can be used to force the hot range
+// the logging task to tick.
+// Within normal operation, there will only be one function listening to this
+// ticker, but in the tests there may be multiple "nodes" within the process.
+// Tests then will need to send multiple requests, to trigger all the nodes.
+var HotRangeLogManualTicker = make(chan struct{}, 0)
```

Comment:
What if we provided a way to mock the ticker being used in tests instead of having both a manual test ticker and the regular one?

Reply:
Hmmmm...not easily, because the channel isn't exposed in a way that an interface can be built around it. Would you prefer it? I can give an example of what it would look like here, or provide an alternative utility.